test/JDBC/expected/alter_proc_recompile-vu-verify.out

```diff
@@ -0,0 +1,323 @@
+-- psql
+analyze master_dbo.tb_alter_recomp_1
+go
+
+-- tsql
+select nspname, funcname, flag_validity&4, flag_values&4 from sys.babelfish_function_ext where funcname like 'p_alter_recomp%' order by funcname
+go
+~~START~~
+varchar#!#varchar#!#bigint#!#bigint
+master_dbo#!#p_alter_recomp_01#!#4#!#0
+master_dbo#!#p_alter_recomp_02#!#4#!#4
+~~END~~
+
+SELECT set_config('babelfishpg_tsql.explain_costs', 'off', false);
+go
+~~START~~
+text
+off
+~~END~~
+
+--==== EXEC with RECOMPILE tests with plan_cache_mode=auto ======
+go
+select set_config('plan_cache_mode', 'auto', false)
+go
+~~START~~
+text
+auto
+~~END~~
+
+select set_config('enable_indexscan', 'on', false)
+go
+~~START~~
+text
+on
+~~END~~
+
+SELECT current_setting('plan_cache_mode') as plan_mode_before
+go
+~~START~~
+text
+auto
+~~END~~
+
+SELECT current_setting('enable_indexscan') as enable_indexscan_before
+go
+~~START~~
+text
+on
+~~END~~
+
+
+--- generating generic plan for p_alter_recomp_01 -----
+go
+declare @i int = 6
+while @i > 0
+begin
+execute p_alter_recomp_01 0  -- created without recompile
+set @i -= 1
+end
+go
+~~START~~
+varchar#!#int#!#int
+~~END~~
+
+~~START~~
+varchar#!#int#!#int
+~~END~~
+
+~~START~~
+varchar#!#int#!#int
+~~END~~
+
+~~START~~
+varchar#!#int#!#int
+~~END~~
+
+~~START~~
+varchar#!#int#!#int
+~~END~~
+
+~~START~~
+varchar#!#int#!#int
+~~END~~
+
+
+--- using index scan ----
+go
+set babelfish_showplan_all on
+go
+execute p_alter_recomp_01 0  -- created without recompile
+go
+~~START~~
+text
+Query Text: EXEC    p_alter_recomp_01 0
+  Query Text: select 'p_alter_recomp_01', * from tb_alter_recomp_1 where c1 = "@p"
+  ->  Index Scan using tb_alter_recomp_1_pkey on tb_alter_recomp_1
+        Index Cond: (c1 = $1)
+~~END~~
+
+execute p_alter_recomp_02 0  -- created with recompile
+go
+~~START~~
+text
+Query Text: EXEC    p_alter_recomp_02 0
+  Query Text: select 'p_alter_recomp_02', * from tb_alter_recomp_1 where c1 = "@p"
+  ->  Index Scan using tb_alter_recomp_1_pkey on tb_alter_recomp_1
+        Index Cond: (c1 = 0)
+~~END~~
+
+set babelfish_showplan_all off
+go
+
+--- turning off index scan ----
+go
+select set_config('enable_indexscan', 'off', false)
+go
+~~START~~
+text
+off
+~~END~~
+
+select current_setting('enable_indexscan') as indexscan_off
+go
+~~START~~
+text
+off
+~~END~~
```

Comment:
We can identify whether recompilation happens or not by just checking the index cond, e.g.,
```
Index Cond: (c1 = $1) --> generic plan (not recompiled)
Index Cond: (c1 = 0) --> custom plan (recompiled)
```
In other words, we don't need to change `enable_indexscan`.

Reply:
removed showplan